valhalla/baldr/pathlocation.h

```diff
@@ -185,7 +174,13 @@ struct PathLocation : public Location {
       stop_type = Location::StopType::VIA;
     else if (loc.type() == odin::Location::kBreakThrough)
       stop_type = Location::StopType::BREAK_THROUGH;
-    Location l({loc.ll().lng(), loc.ll().lat()}, stop_type, loc.minimum_reachability(), loc.radius());
+    auto side = PreferredSide::EITHER;
+    if (loc.preferred_side() == odin::Location::same)
+      side = PreferredSide::SAME;
+    else if (loc.preferred_side() == odin::Location::opposite)
+      side = PreferredSide::OPPOSITE;
+    Location l({loc.ll().lng(), loc.ll().lat()}, stop_type, loc.minimum_reachability(), loc.radius(),
+               side);
```

Comment:
bug, the tests didnt test this so it was skipped. now they do, and they failed when it wasnt here.